include/micm/solver/rosenbrock_solver_parameters.hpp

```diff
@@ -51,7 +52,7 @@ namespace micm
     // Gamma_i = \sum_j  gamma_{i,j}
     std::array<double, 6> gamma_{};
 
-    double absolute_tolerance_{ 1e-3 };
+    std::vector<double> absolute_tolerance_;
```

Comment:
Is there a reason to set `absolute_tolerance` individually for each species but an overall `relative_tolerance` for everything?

Reply:
Only that the camp config doesn't yet support that.

Reply:
Thanks for the information.